### PR TITLE
Add text in jugaad when no files present

### PR DESCRIPTION
**Pull request by erilyth**
T73

### CODE REVIEW

**Comment by nisargjhaveri** [comment]
We don't really need to use Javascript here. Just check the variable containing file list and if empty echo the warning from server itself.

**Comment by nisargjhaveri** [comment]
Logically, looks fine. Someone check indentation and html quality (uses a br). Indentation looks weired for me on mobile so please check. Not tested.

**Comment by erilyth** [comment]
Swapped out the br tag with a p tag. Also adjusted the indentation a bit.

**Comment by nisargjhaveri** [comment]
Still something wrong when I check changes for the PR in mobile. 

Anyway, squash and ask someone else to test and merge.

**Comment by polybuildr** [comment]
It's a tabs vs spaces issue. @erilyth we're using 4 spaces as a convention for indentation, you've used tabs.

File: src/app/views/directory.php [hunk 1]
```diff
@@ -44,8 +49,8 @@
             </article>
         </div>
         <script>
-        var links = document.querySelectorAll('.item-list a');
-        for (i = 0; i < links.length; ++i) {
+	var links = document.querySelectorAll('.item-list a');
```
Comment (polybuildr) [inline src/app/views/directory.php]: Were these 2 lines supposed to be changed in this PR? (Again indentation is changed, that's all.)

File: src/app/views/directory.php [hunk 2]
```diff
@@ -19,7 +19,12 @@
                     <?php if ($id == 0 && $user_can['see_global_trash']): ?>
                         <a href="<?= base_url() ?>trash/" class="btn btn-green"><i class="fa fa-trash"></i> Go to trash</a>
                     <?php endif; ?>
-                </div>
+		</div>
+		<?php 
+		    if (empty($data)) {
+			echo "<p>No files/directories are currently present, please check back later.</p>";
```
Comment (polybuildr) [inline src/app/views/directory.php]: This sounds like this is a blog and we're giving updates. :P There's no guarantee that someone will ever be putting up something, so the "check back later" part seems a little weird, don't you think?

**Comment by erilyth** [comment]
Squashed and fixed tabs issue.

**Comment by polybuildr** [comment]
LGTM. Merging.

**Pull request merged**